src/instructlab/defaults.py

```diff
@@ -174,6 +177,10 @@ def DEFAULT_CHAT_MODEL(self) -> str:
     def DEFAULT_EMBEDDING_MODEL(self) -> str:
         return path.join(self.MODELS_DIR, self.GRANITE_EMBEDDING_MODEL_NAME)
 
+    @property
+    def DEFAULT_DOCUMENT_STORE_PATH(self) -> str:
```

Comment:
we might want to add this to `ensure_storage_directories_exist`, which is a func that creates all necessary directories before running any ilab cmd

Reply:
+1, not blocking as we can do this in one of the other RAG PRs as well

Reply:
`DEFAULTS._data_dir` is already in `ensure_storage_directories_exist`; the `DOCUMENT_STORE_NAME` is a filename:

```
DOCUMENT_STORE_NAME = "embeddings.db"
```